patroni/ctl.py

```diff
@@ -800,3 +805,41 @@ def flush(cluster_name, member_names, config_file, dcs, force, role, target):
                 check_response(r, member.name, 'flush scheduled restart')
             else:
                 click.echo('No scheduled restart for member {0}'.format(member.name))
+
+
+@ctl.command('disable', help='Disable auto failover')
+@click.argument('cluster_name')
+@option_config_file
+@option_dcs
+def disable(config_file, cluster_name, dcs):
```

Comment:
Should the following 2 functions be merged into one (i.e. toggle_pause(config_file, cluster_name, dcs, pause_state), with pause calling toggle_pause(..., True) and resume calling toggle_pause(..., False). It may simplify the tests as well, less lines to cover :-)